src/store/middleware/emergencyStopMiddleware.js

```diff
@@ -18,8 +17,6 @@ const emergencyStopMiddleware = store => next => action => {
           stop: store.getState().emergencyStop.stopped
         }
       }));
-      // send ik packet with false
-      store.dispatch(enableIK({ enable: false }));
```

Comment:
Don't think these changes should be on this PR? Did you need to rebase?

Reply:
I meant to do this in the other PR, but since the other one's merged already I'm going to keep this here.